Documentation/tpv13.rst

```diff
@@ -3,58 +3,49 @@
 SCEC TPV13
 ==========
 
-TPV13 is similar to TPV12 except for that material properties are **non-associative Drucker-Prager plastic**. To run TPV13, it requires to recompile SeisSol with "-DPLASTICITY=ON" in the configuration python script. 
+TPV13 is similar to TPV12 except for the **non-associative Drucker-Prager visco-plastic** rheology.  
 
-The material is characterized by six constitutive parameters:
+The Drucker-Prager yield function is given by:
 
- *Bulk friction = 0.85*
-  *Fluid pressure = 1000 kg/m3*
-  Based on the SCEC benchmark, the stress components
-  
-  :math:`\sigma_{ij}` are defined as :
-  
-  *Mean stress*:
-  
-  :math:`\sigma_m = (\sigma_{11}+\sigma_{22}+\sigma_{33})/3`
-  
-  Stress deviator: :math:`s_{ij} = \sigma_{ij} - \sigma_m \delta_{ij}`
-  Second invariant of the stress deviator:
-  
-  :math:`J_2(\sigma) = 1/2 *\sum_{ij} s_{ij} s_{ji}`
-  
-  Drucker-Prager yield stress:
+  :math:`F(\sigma)=\sqrt{J_2(\sigma)-Y(\sigma)}`
+
+:math:`Y(\sigma)` is the  Drucker-Prager yield stress, given as:
   
   :math:`Y(\sigma) =\max(0,c\cos \phi - (\sigma_m +P_f)\sin \phi)`
-  
-  Drucker-Prager yield function:
-  
-  :math:`F(\sigma)=\sqrt{J_s(\sigma)-Y(\sigma)}`
 
-The Drucker-Prager material is required to satisfy the yield equation:
+with  :math:`\sigma_m = (\sigma_{11}+\sigma_{22}+\sigma_{33})/3` the mean stress,
+
+:math:`c` the bulk cohesion, :math:`\phi` the bulk friction and :math:`P_f` the fluid pressure (1000 kg/m :math:`^3`).
+In TPV13 benchmark, :math:`c=` 5.0e+06 Pa and :math:`\phi` =0.85.
+
+:math:`J_2` is the second invariant of the stress deviator:
+
+  :math:`J_2(\sigma) = 1/2 \sum_{ij} s_{ij} s_{ji}`
+
+with :math:`s_{ij} = \sigma_{ij} - \sigma_m \delta_{ij}` the deviator stress components.
+
+
+The yield equation has to be satisfied:
 
   :math:`F(\sigma)\leq 0`
-  
+   
 When :math:`F(\sigma) < 0`, the material behaves like a linear isotropic elastic material, 
 with Lame parameters :math:`\lambda` and  :math:`\mu`.
 
 Wen :math:`F(\sigma) = 0`, if the material is subjected to a strain that 
 tends to cause an increase in :math:`F(\sigma)`, then the material
-yields. For TPV13, we assume that the material yields in shear. Yielding
-in shear means that when the material yields, the stress tensor
-:math:`\sigma_{ij}` changes by an amount proportional to the stress
-deviator :math:`s_{ij}`, so as to preserve the condition
-:math:`F(\sigma)` with no change in mean stress :math:`\sigma_m` .
+yields and plastic strains accumulates.
 
 Nucleation
 ~~~~~~~~~~
 
-TPV13 uses the same nucleation method as TPV12
+TPV13 uses the same nucleation strategy as TPV12.
 
 Plasticity parameters
 ~~~~~~~~~~~~~~~~~~~~~
 
 To turn on plasticity in SeisSol, add the following lines in
-*parameter.par* (https://github.com/SeisSol/Examples/blob/master/tpv12_13/parameters.par):
+*parameters.par* (https://github.com/SeisSol/Examples/blob/master/tpv12_13/parameters.par):
 
 .. code-block:: Fortran
   
```

Comment:
```
.. code-block:: Fortran

  &SourceType
  Plasticity = 1 ! default = 0
  Tv = 0.03 ! Plastic relaxation
  /
```

Should it be removed? Or `SourceType` needs to be replaced with `Equations`?

Reply:
you are right, I did not notice that.